docs/components/navigation.tsx

```diff
@@ -14,6 +14,17 @@ export default class Navigation extends React.Component<IProps> {
     return <div className={styles.navigation}>{this.renderTrees(this.props.tree.children)}</div>;
   }
   private renderTrees(trees: IFileTree[], depth = 0) {
+    // If there's a priority, order by priority. If not, order alphabetically.
+    trees.sort((a: IFileTree, b: IFileTree) =>
+      a.attributes.priority == null || b.attributes.priority == null
+        ? a.attributes.title > b.attributes.title
```

Comment:
use a.compareTo(b) for strings

Reply:
Lmk if I'm wrong, but that's Java. In JS the `>` operator is the right way to compare strings, unless wanting a browser specific check with [localeCompare()](https://www.w3schools.com/jsref/jsref_localecompare.asp).

Reply:
localCompare is what I meant yeah but But I think you are right